internal/server/kong/ws/config/compat/compatibility_table.go

```diff
@@ -834,6 +842,20 @@ var (
 				},
 			},
 		},
+		{
+			Metadata: config.ChangeMetadata{
+				ID:          config.ChangeID("P135"),
+				Severity:    config.ChangeSeverityError,
+				Description: standardCoreEntityMessage("vault", "3.0"),
+				Resolution:  standardUpgradeMessage(versions300AndAbove),
```

Comment:
```suggestion
				Resolution:  standardUpgradeMessage("3.0"),
```

Reply:
Otherwise this gets rendered like this:

`Please upgrade Kong Gateway to version '>= 3.0.0' or above.`

Reply:
fixed this